partitioned-table.md

```diff
@@ -560,6 +560,15 @@ MOD(YEAR('2005-09-01'),4)
 =  1
 ```
 
+#### How TiDB handles LINEAR HASH partitions
+
+Before v6.4.0, if you execute DDL statements of [MySQL Linear Hash](https://dev.mysql.com/doc/refman/5.7/en/partitioning-linear-hash.html) partitions in TiDB, TiDB can only create non-partitioned tables. In this case, if you still want to use partitioned tables in TiDB, you need to modify the DDL statements.
+
+Since v6.4.0, TiDB supports parsing the MySQL `PARTITION BY LINEAR HASH` syntax but ignores the `LINEAR` keyword in it. If you have some existing DDL and DML statements of MySQL Linear Hash partitions, you can execute them in TiDB without modification:
+
+- For DDL statements of MySQL Linear Hash partitions, TiDB will create non-linear Hash partitioned tables correspondingly (note that Linear Hash partitioned tables are not provided in TiDB). 
+- For DML statements of MySQL Linear Hash partitions, as long as you do not use [partition selection](#partition-selection), TiDB will return the query results of the corresponding TiDB Hash partitioned normally.
```

Comment:
```suggestion
- For DML statements of MySQL Linear Hash partitions, as long as you do not use [partition selection](#partition-selection), TiDB will return the query results of the corresponding TiDB Hash partitioned normally.  For explicit partition selection, if the number of partitions is a multiple of 2 (which means that the rows of a table can be distributed evenly among the partitions), TiDB can return the results as expected. Otherwise, a partition will be folded into other partitions and TiDB will return a different set of rows.
```

Reply:
@mjonss @moyun I added a note for the case of partition selection, PTAL. Thanks.

Reply:
LGTM

Reply:
@qiancai How about the change I proposed below?

Reply:
OK. Looks good to me. Let me commit the change and refine some wording.